.github/workflows/benchmarks_report.yml

```diff
@@ -18,6 +18,7 @@ on:
 jobs:
   download:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run }}
```

Comment:
What does this line do in this context? It might be worth adding a comment to it...

Reply:
Will remove it, as it is connected with try to fix reporting. I will make followup PR for reporting